context/MqlContext/MqlContextProvider.tsx

```diff
@@ -139,11 +139,9 @@ function MqlContextProviderInternal({
     handleCombinedError,
   });
 
-
-
   useEffect(() => {
     const stateToUpdate: Partial<MqlContextType> = {};
-    const useOverride = !!mqlServerUrlOverride;
+    const useOverride = Boolean(mqlServerUrlOverride);
```

Comment:
It seems to me that we don't really need all this logic about overrides. Hopefully the backend API abstracts this logic away. Though that can probably wait until we have more time to rip it out

Reply:
Yeah. In this case I believe it's to pass in a value via an environment variable, if you want to override it without hitting the backend.

Reply:
It does feel needlessly complex, though!

Reply:
This is def. more complex than it needs to be. I may revisit this today before deploying a new version of the package.